### DISCUSSION TITLE
RESOLVED Feature request: make it possible for vendors to disable maintenance page in config file

### DISCUSSION THREAD

**Discussion by Kixunil** [btcpayserver/btcpayserver#2526] [Technical Support] [answered]
**Problem**
There are several alternative vendors (including myself) providing BTCPayServer outside of the upstream docker deployment. The maintenance page is not relevant for them and confuses users.

**Proposed solution**
Make it possible to hide the maintenance page in config **file**, which the vendors will set. Upstream docker keeps it visible.

**Alternatives**
* The page may be removed at build time
* Add disclaimer text that it only applies to the upstream docker deployment

**Comment by Kukks** [comment, marked as answer]
Already exists, please see https://github.com/btcpayserver/btcpayserver-docker/blob/5fec2aa642593788c3ce053a1dd77c040b894d5d/docker-compose-generator/docker-fragments/btcpayserver.yml#L22 and set it to false

**Reply by Kixunil** [reply]
Awesome, thanks!

**Reply by Kixunil** [reply]
Confirmed, it works. :)
